src/utils/invoice.js

```diff
@@ -47,11 +45,19 @@ export function fetchLnurl(lnurl, amount_sat) {
     });
 }
 
-export function trimLightningPrefix(invoice) {
-    if (invoice.toLowerCase().startsWith(invoicePrefix)) {
-        return invoice.slice(invoicePrefix.length);
+export function trimPrefix(invoice, prefix) {
+    if (invoice.toLowerCase().startsWith(prefix)) {
+        const url = new URL(invoice);
+        return url.pathname;
```

Comment:
And there is no test making sure the URL query parameters of the `invoice` variable are removed